packages/ui-library/src/components/transaction-receipt/index.tsx

```diff
@@ -721,7 +716,7 @@ const TransactionReceipt = ({ transaction, open, onClose }: TransactionReceiptPr
   return (
     <StyledDialog open={open} scroll="paper" maxWidth="xs" fullWidth PaperProps={{ id: 'paper-id' }} onClose={onClose}>
       <StyledDialogTitle>
-        {icon}
+        <BalmyLogoSmallLight size={spacing(8)} />
```

Comment:
What about dark theme?

Reply:
The receipt icon is always the light one
<img width="465" alt="Screenshot 2024-02-21 at 13 49 27" src="https://github.com/Mean-Finance/dca-fe/assets/144699868/1c5cd65d-a9a9-49df-a033-416cce592971">
<img width="494" alt="Screenshot 2024-02-21 at 13 49 44" src="https://github.com/Mean-Finance/dca-fe/assets/144699868/901a02d0-ce01-45a4-ba75-a6b74f86f132">